packages/dataviews/src/types.ts

```diff
@@ -368,7 +356,13 @@ export interface ActionButton< Item extends AnyItem >
 	/**
 	 * The callback to execute when the action is triggered.
 	 */
-	callback: ( items: Item[] ) => void;
+	callback: (
+		items: Item[],
+		context: {
+			registry: any;
+			onActionPerformed?: ( items: Item[] ) => void;
+		}
```

Comment:
Is the `onActionPerformed` field used anywhere? I see that all calls pass just `{ registry }`.

Is it possible to make `context` the first argument? It's something like `state` for a selector or a `this` argument when defining type for a TS function. It will look odd when you later add a third or four parameter and `context` will be randomly somewhere in the middle.

Reply:
I think it makes more sense to put the context as the last argument since it's not really necessary for the callback. The "items" is more important. 

Also, one of the reasons, I made it an object is to be able to add more optional configs later there.

For the onActionPerformed, yes it's used in both post and site editors (but not in dataviews). I don't love this API personally, but I don't want to change everything here. It can potentially be made unnecessary in callback actions by making these functions return promises but it will be necessary in Modal Actions as a prop, so maybe it's fine to keep it for callback actions as an argument too.

Reply:
The downside is that if you ever add new parameters (how likely is that?) then `context` will have to move or will end up in a weird position. 🤷‍♂️